src/components/TreeFilter/TreeFilter.scss

```diff
@@ -0,0 +1,106 @@
+@import '../../index.scss';
+
+ul {
+    margin: 0;
+    list-style: none;
+}
+
+.item {
+    height: 20px;
+    display: inline-block;
+    flex-direction: column;
+    justify-content: center;
+    white-space: pre;
+    user-select: none;
+}
+
+.item-leaf{
+    height: 20px;
+    span{
+        padding-left: 20px;
+    }
+}
+
+.tree-expand-icon{
+    vertical-align: middle;
+}
+
+.tree-filter-input{
+    width: calc(100% - 5px);
+}
+
+.tree-filter-container {
+    width: 200px;
+    height: 20px;
+    padding: 2px 1px 2px 1px;
+    border-bottom: 1px solid $secondary-color;
+    cursor: pointer;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+}
+
+.tree-single-select-item{
+    display: inline-block;
+    color: $font-primary-color;
+    font-family: $font-family-base;
+    font-size: $font-size-m;
+}
+
+// checkbox
+$Checkbox-label-size: 20px;
+$Checkbox-transition-duration: 200ms;
+$Checkbox-transition-timing: cubic-bezier(.4, 0, .23, 1);
+
+.checkbox {
```

Comment:
the style was copied from the checkbox component. I have changed the class name to avoid conflict